File: GTTS/INL.TrainingService.Models/GetTrainingEventKeyActivity_Item.cs
using System;

namespace INL.TrainingService.Models
{
    public class GetTrainingEventKeyActivity_Item : IGetTrainingEventKeyActivity_Item
    {
        public int KeyActivityID { get; set; }
        public long TrainingEventID { get; set; }
        public string Code { get; set; }
        public string Description { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public DateTime ModifiedDate { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-training-event-key-activity_item.ts


import { IGetTrainingEventKeyActivity_Item } from './iget-training-event-key-activity_item';

export class GetTrainingEventKeyActivity_Item implements IGetTrainingEventKeyActivity_Item {
  
	public KeyActivityID: number = 0;
	public TrainingEventID: number = 0;
	public Code: string = "";
	public Description: string = "";
	public ModifiedByAppUserID: number = 0;
	public ModifiedDate: Date = new Date(0);
  
}






